.github/workflows/e2e-init-chain.yml

```
@@ -0,0 +1,54 @@
+name: Build and Push E2E Init Chain Images
+
+on:
+  push:
+    tags:
+    - 'v12+.[0-20]+.0+' 
+    - 'v[0-20]+.0+.0+' 
```

```suggestion
    - 'v[0-9]+.[0-9]+.0'
```

test: https://regex101.com/r/bx6RJY/1